physical/raft/fsm.go

```diff
@@ -653,3 +702,149 @@ func protoConfigurationToRaftConfiguration(configuration *ConfigurationValue) (u
 		Servers: servers,
 	}
 }
+
+type FSMChunkStorage struct {
+	f   *FSM
+	ctx context.Context
+}
+
+// chunkPaths returns a disk prefix and key given chunkinfo
+func (f *FSMChunkStorage) chunkPaths(chunk *raftchunking.ChunkInfo) (string, string) {
+	prefix := fmt.Sprintf("%s%d/", chunkingPrefix, chunk.OpNum)
+	key := fmt.Sprintf("%s%d-%d", prefix, chunk.NumChunks, chunk.SequenceNum)
```

Comment:
I'm not sure it's necessary to encode the total numChunks here, see comment below

Reply:
Fixed.